AUTHORS.md

```diff
@@ -0,0 +1,45 @@
+# Credits
+
+---
+© 2024 **xdem developers**.
```

Comment:
If this applies here: For use in plain text purposes, we had previously converged on the writing "xDEM" (currently consistent nearly everywhere in documentation, README, etc).

Happy to re-discuss that syntax if you have an opinion! @duboise-cnes @adebardo @vschaffn
Also tagging @adehecq @erikmannerfelt who were part of the initial discussion.
In short our logic was:
1. We want to have upper-case where relevant in the name, same as "NumPy", "SciPy", etc (not the case for all package, like for "pandas", but usually is),
2. We chose to keep the DEM upper-case to clarify its acronym origin.

The advantage of that writing is that makes it obvious that it's about DEMs (which was our priority), while some disadvantages are that it starts with a lower-case (slightly atypical) and is not consistent with the writing of "Xarray" (not a big worry).

Not a super important topic, it's more a question of style, but we might as well get it right directly in the new License with everyone onboard! :slightly_smiling_face:

Reply:
as you see fit. xDEM everywhere perfect for me.